app/helpers/rubygems_helper.rb

```diff
@@ -139,6 +139,14 @@ def link_to_github(rubygem)
 
   def github_params(rubygem)
     link = link_to_github(rubygem)
-    "user=#{link.path.split('/').second}&repo=#{link.path.split('/').third}&type=star&count=true&size=large" if link
+    return unless link
```

Comment:
Should we also take into consideration an empty string as well?

```suggestion
    return unless link.present?
```